content/en/docs/reference/access-authn-authz/controlling-access.md

```diff
@@ -111,9 +111,9 @@ To learn more about Kubernetes authorization, including details about creating p
 Admission Control Modules are software modules that can modify or reject requests.
 In addition to the attributes available to Authorization Modules, Admission
 Control Modules can access the contents of the object that is being created or updated.
-They act on objects being created, deleted, updated or connected (proxy), but not reads.
-
-Multiple admission controllers can be configured.  Each is called in order.
+They act on objects being created, deleted, updated or connected (proxy), but not reads. They can
+be applied by 4 verbs: create, delete, update, and connect. When multiple admission controllers
+are being configured, each is called in order.
```

Comment:
```suggestion
are configured, each admission controller is called in order.
```

I'm also not clear on what order they are called in? (I understand it's also not clear in the original docs either)

Reply:
Thanks @jimangel . I'll look into the code and see if we can document it this way.